code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -87,17 +90,46 @@
 		beaker = null
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/update_icon()
+	cut_overlays()
+
 	if(panel_open)
-		icon_state = "cell-o"
-	else if(state_open)
-		icon_state = "cell-open"
+		icon_state = "pod-o"
 	else if(on && is_operational())
-		if(occupant)
-			icon_state = "cell-occupied"
-		else
-			icon_state = "cell-on"
+		icon_state = "pod-on"
 	else
-		icon_state = "cell-off"
+		icon_state = "pod-off"
+
+	if(occupant)
+		occupant_overlay = mutable_appearance(occupant.icon, occupant.icon_state)
+		occupant_overlay.overlays = occupant.overlays
+		occupant_overlay.pixel_y = 22
+		add_overlay(occupant_overlay)
+		if(on && is_operational() && !bobbing)
+			var/bobup = TRUE
+			spawn(0) // Time to bob

```

Comment:
Make this a seperate proc like /obj/machinery/atmospherics/components/unary/cryo_cell/proc/cryo_animation